### PR TITLE
Dev/ft/ring 28536 add node clinic

### PR DESCRIPTION
**Pull request by maximelubin**
Added nodejs performance diagnostic tool: node-clinic
Link:https://github.com/nearform/node-clinic

Capable of inspecting CPU, memory, GC, event-loop and I/O issues, generating flame graphs.
Several examples have been added in the README.

NB: requires nodejs > 8.X so must wait for [this pull request](https://github.com/scality/hdclient/pull/10) to be merged.

As a teaser, you can find below a flame graph: 5K PUT of 32K, ECN2+1 with 64 parallel worker, followed by a GET on all keys, all other things being equal.
 
NB: done with Benoit's sproxydbench tool

put_get_flame_graph

### CODE REVIEW

**Comment by maximelubin** [comment]
@bert-e help

File: README.md [hunk 1]
```diff
@@ -61,12 +63,44 @@ npm test tests/unit/
...
 
+```shell
+# Node clinic diagnosis tool - help
+npm run clinic
+
+# Clinic doctor help
+npm run clinic doctor
+
+# Diagnosing on the same machine - use --on-port to kickstart load generator
+NODE_ENV=roduction npm run clinic doctor -- \
```
Comment (ghost) [inline README.md]: production* maybe ?

File: README.md [hunk 2]
```diff
@@ -61,12 +63,44 @@ npm test tests/unit/
...
+                   --on-port='for i in {..10}; do curl -XPUT --data-binary @/etc/hosts "http://localhost:6767/bucket/testobj$i" ; done' \
+                   -- node scripts/server.js 6767 scripts/example_hdclient_proxy.conf.json
+
+NODE_ENV=roduction npm run clinic doctor -- \
```
Comment (ghost) [inline README.md]: idem, p is missing

File: README.md (hunk 1 above)
Comment (maximelubin) [reply README.md]: Indeed :-) Fixed

**Review by lunatikub — DISMISSED** [review]

**Review by lunatikub — APPROVED** [review]

**Review by xroche-scality — APPROVED** [review]
Are you sure you want to keep the <<< and >>> from conflict here ? They seem to be removed later, but yet...

**Review by xroche-scality — APPROVED** [review]

**Review by xroche-scality — APPROVED** [review]

**Review by xroche-scality — APPROVED** [review]

**Review by xroche-scality — APPROVED** [review]

**Review by cstoyanov-scality — APPROVED** [review]

**Pull request merged**